From 08e396f5827147988e417ef148872ac2381c3843 Mon Sep 17 00:00:00 2001
From: "gregkh@suse.de" <gregkh@suse.de>
Date: Tue, 20 Sep 2011 09:56:46 -0700
Subject: [PATCH 70/91] Patch "acpica: ACPI_MAX_SLEEP should be 2 sec, not 20"
 has been added to the 3.0-stable tree

This is a note to let you know that I've just added the patch titled

    acpica: ACPI_MAX_SLEEP should be 2 sec, not 20

to the 3.0-stable tree which can be found at:
    http://www.kernel.org/git/?p=linux/kernel/git/stable/stable-queue.git;a=summary

The filename of the patch is:
     acpica-acpi_max_sleep-should-be-2-sec-not-20.patch
and it can be found in the queue-3.0 subdirectory.

If you, or anyone else, feels it should not be added to the stable tree,
please let <stable@kernel.org> know about it.

>From b33c25d6a62ac253caabda2b5f43258abff451c0 Mon Sep 17 00:00:00 2001
From: Len Brown <len.brown@intel.com>
Date: Mon, 29 Aug 2011 23:01:58 -0400
Subject: acpica: ACPI_MAX_SLEEP should be 2 sec, not 20

From: Len Brown <len.brown@intel.com>

commit b33c25d6a62ac253caabda2b5f43258abff451c0 upstream.

This limit is a workaround for AML that sleeps too long,
but the workaround didn't work b/c of a typo.

https://bugzilla.kernel.org/show_bug.cgi?id=13195

Signed-off-by: Len Brown <len.brown@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/acpi/acpica/acconfig.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/drivers/acpi/acpica/acconfig.h
+++ b/drivers/acpi/acpica/acconfig.h
@@ -121,7 +121,7 @@
 
 /* Maximum sleep allowed via Sleep() operator */
 
-#define ACPI_MAX_SLEEP                  20000	/* Two seconds */
+#define ACPI_MAX_SLEEP                  2000	/* Two seconds */
 
 /******************************************************************************
  *
